make-release.sh

```diff
@@ -302,13 +325,20 @@ verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-machine-exec:${
 verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-devfile-registry:${CHE_VERSION} 60
 # shellcheck disable=SC2086
 verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-dashboard:${CHE_VERSION} 60
+# shellcheck disable=SC2086
+verifyBranchExistsWithTimeout "https://github.com/che-incubator/configbump.git" ${BRANCH}
```

Comment:
you aren't passing in a timeout value... so... this will likely fail because `this_timeout` will either be `''` or `0`. And if `$count` is not `-le` `$timeout_intervals` then the while loop will never run, so you'll get an error immediately.

Reply:
thanks, added the missing timeouts